ui/src/components/home/ProfileContainerDesktop.tsx

```diff
@@ -0,0 +1,387 @@
+            WebkitTextFillColor: 'black',
+          },
+        }}
+      />
+      <Typography variant="h3" fontWeight={400}>
+        Email
+      </Typography>
+      <Divider style={{ marginBottom: '3%' }} />
+      <TextField
+        fullWidth
+        disabled
+        id="outlined-email"
+        label="Email"
+        {...props.register('email', { required: true })}
+        style={{ marginBottom: '5%' }}
+        sx={{
+          '& .MuiInputBase-input.Mui-disabled': {
+            WebkitTextFillColor: 'black',
+          },
+        }}
+      />
+      <Button variant="contained" onClick={props.onClickEdit}>
+        <Typography variant="subtitle2">EDIT PROFILE</Typography>
+      </Button>
+      <Button variant="contained" onClick={props.onClickChange}>
+        <Typography variant="subtitle2">CHANGE PASSWORD</Typography>
+      </Button>
+    </Stack>
+  );
+}
+
+function EditProfile(props: any) {
+  return (
+    <Stack spacing={2} width="40%" margin={5}>
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          flexWrap: 'wrap',
+        }}
+      >
+        <ArrowBackIosNewIcon onClick={props.onBack} sx={{ color: 'gray' }} />
+        <Typography variant="h3" color="primary" fontWeight="bold">
+          EDIT PROFILE
+        </Typography>
+        <CreateIcon color="primary" style={{ marginLeft: '2%' }} />
+      </div>
+      <Typography variant="h3" fontWeight={400}>
+        Old Name & Email
+      </Typography>
+      <Divider style={{ marginBottom: '3%' }} />
+      <TextField
+        fullWidth
+        disabled
+        id="outlined-name"
+        label="Name"
+        {...props.register('fullName', { required: true })}
+        style={{ marginBottom: '5%' }}
+      />
+      <TextField
+        fullWidth
+        disabled
+        id="outlined-email"
+        label="Email"
+        {...props.register('email', { required: true })}
+        style={{ marginBottom: '5%' }}
+      />
+      <Typography variant="h3" fontWeight={400}>
+        New Name
+      </Typography>
+      <Divider style={{ marginBottom: '3%' }} />
+      <form onSubmit={props.onSubmit} style={{ marginBottom: '10%' }}>
+        <TextField
+          fullWidth
+          id="outlined-name"
+          {...props.register('fullName')}
+          label="Enter New Name"
+          style={{ marginBottom: '5%' }}
+        />
+        <TextField
+          fullWidth
+          disabled
+          id="outlined-email"
+          label="Email"
+          {...props.register('email', { required: true })}
+          style={{ marginBottom: '5%' }}
+          sx={{
+            '& .MuiInputBase-input.Mui-disabled': {
+              WebkitTextFillColor: 'black',
+            },
+          }}
+        />
+        <Button variant="contained" type="submit" fullWidth>
+          <Typography variant="subtitle2">SAVE</Typography>
+        </Button>
+      </form>
+    </Stack>
+  );
+}
+
+function ChangePassword(props: any) {
```

Comment:
If u hav time, u can also refactor some of this code to a new file so its not one huge file.

Reply:
it's gonna be a bunch of files cuz the functions are diff for mobile and dekstop soooi decided to just put it all in one